### ISSUE TITLE
.NET Core 3.0 RC1

### ISSUE THREAD

**Issue by leecow** [dotnet/core#3380]
[Release Notes](https://github.com/dotnet/core/blob/master/release-notes/3.0/preview/3.0.0-rc1.md)
[Download](https://github.com/dotnet/core/blob/master/release-notes/3.0/preview/3.0.0-rc1-download.md)

**Blog posts**

[.NET Core](https://devblogs.microsoft.com/dotnet/announcing-net-core-3-0-release-candidate-1/)
[ASP.NET Core](https://devblogs.microsoft.com/aspnet/asp-net-core-and-blazor-updates-in-net-core-3-0-release-candidate-1/)

Please report any issues you find with 3.0.0 RC1, either responding to this issue, creating a new issue or creating a new issue in one of the following repos:

* [aspnet/AspNetCore](https://github.com/aspnet/AspNetCore) - for ASP.NET Core questions
* [aspnet/EntityFrameworkCore](https://github.com/aspnet/EntityFrameworkCore) - for Entity Framework Core questions
* [dotnet/cli](https://github.com/dotnet/cli/issues) - for CLI tools and questions
* [dotnet/corefx](https://github.com/dotnet/corefx/issues) - for API issues and questions
* [dotnet/coreclr](https://github.com/dotnet/coreclr/issues) - for runtime issues
* [dotnet/core-setup](https://github.com/dotnet/core-setup/issues) - for installer issues
* [dotnet/winforms](https://github.com/dotnet/winforms/issues) - for Winforms issues
* [dotnet/wpf](https://github.com/dotnet/wpf/issues) - for WPF issues
* [nuget/home](https://github.com/nuget/home/issues) - for NuGet questions and issues

**Comment by smjltd** [comment]
how to create blazor client side this below command not working

`dotnet new blazorwasm -o WebApplication1`

**Comment by ddgweb** [comment]
httpclient or httpwebrequest
The same code ,for windows +ngix no error , everythiny is ok , bu in linux centos, error tips:
The SSL connection could not be established, see inner exception. The remote certificate is invalid according to the validation procedure.

I already set "http.RemoteCertificateValidationCallback += (sender, certificate, chain, errors) => true;" but it's don't work

**Comment by akaylh** [comment]
@ddgweb try this:

solution 1:

```cs
services.AddHttpClient<T>()
        .ConfigurePrimaryHttpMessageHandler(() =>
        {
            return new HttpClientHandler
            {
                ServerCertificateCustomValidationCallback = (m, c, c, e) => true
            };
        });
```

solution 2

```cs
using (var httpClientHandler = new HttpClientHandler())
{
   httpClientHandler.ServerCertificateCustomValidationCallback = (message, cert, chain, errors) => { return true; };
   using (var client = new HttpClient(httpClientHandler))
   {
       // Make your request...
   }
}
```

**Comment by Postlagerkarte** [comment]
Using the RC 1 still prints on the build output "You are using a preview version of .NET Core" - even though in the release notes it says "NET Core 3.0 Preview RC1 is supported by Microsoft and can be used in production."

**Comment by ddgweb** [comment]
> @ddgweb try this:
> 
> solution 1:
> 
> ```cs
> services.AddHttpClient<TClient>()
>         .ConfigurePrimaryHttpMessageHandler(() =>
>         {
>             return new HttpClientHandler
>             {
>                 ServerCertificateCustomValidationCallback = (m, c, c, e) => true
>             };
>         });
> ```
> 
> TClient is your [typed client](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/http-requests?view=aspnetcore-2.2#typed-clients)
> 
> solution 2
> 
> ```cs
> using (var httpClientHandler = new HttpClientHandler())
> {
>    httpClientHandler.ServerCertificateCustomValidationCallback = (message, cert, chain, errors) => { return true; };
>    using (var client = new HttpClient(httpClientHandler))
>    {
>        // Make your request...
>    }
> }
> ```

I tried,  it don't work , and error  upgrade "An error occurred while sending the request. The server returned an invalid or unrecognized response."

the same code, on windows server anything is ok , I can't find the problem

**Comment by MarcusWichelmann** [comment]
Do you have an APT repository for this release candidate? Or will you push the final release to the main repository direcly after the keynote?

**Comment by petermcclymont** [comment]
I installed .NET Core 3.0 RC1 on Kubuntu 19.10.

My app doesn't work in this scenario, so I decided to go back to basics. Create a new console app to see what would happen,

mkdir testdotnet
cd testdotnet
dotnet new console
dotnet run

It runs for a second, and gives nothing. No output, whereas I would normally expect,

Hello World!

I am using a snap package, and if I roll back to .NET Core 2.2 it works.

What to do next? Is there some kind of verbose output or debugging that I can use to help understand why this is not working?
